public/index.tpl

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Wobscale.chat password reset</title>
+    <script>
+      
+    </script>
+  </head>
+  <body>
+    <h1>So, you forgot your password</h1>
+
+    <h2> {{ notice }} </h2>
```

Comment:
Is this an empty `h2` if `notice` isn't defined? Although I don't know mustache that well, so I'm not sure if it's possible to remove the tag if it's not defined.